README.md

```diff
@@ -1,21 +1,21 @@
-# Welcome to KMS!
+# Welcome to Citadel!
 
-KMS is a simple daemon that implements the Kubernetes Key Management Service
+Citadel (c5l) is a simple daemon that implements the Kubernetes Key Management Service (KMS)
```

Comment:
I don't like the `c5l` abbreviation. `citadel` isn't that long.

Reply:
I used it as the short name in the prefix code (to mimic kube), so I wanted to actually use it so people would know what it was referring to.

I am not really worried about the length in the docs, its more of "google will find it"

Reply:
I still don't like it, but if you want to mimic kube, I can accept that.